### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by binarykitchen** [groundwater/node-stackup#6]
Yeah, I see that in my error.log of my app:

```
$ cat error.log
/var/www/videomail/production/source/node_modules/stackup/trace.js:13
  if (this.stack)  out.push(this.__format(this.stack.stack));
                                                    ^
RangeError: Maximum call stack size exceeded
    at join (native)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:13:53)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
    at Trace.toString (/var/www/videomail/production/source/node_modules/stackup/trace.js:14:41)
```

**Comment by hayes** [comment]
this should be fixed by #7

**Comment by binarykitchen** [comment]
thanks. let me know when it's out, on npm ...

**Comment by hayes** [comment]
fixed in v1.0.1

**Comment by binarykitchen** [comment]
thx man
